libs/dh/admin/feature-invite-user-modal/src/lib/dh-assignable-user-roles/dh-assignable-user-roles.component.scss

```diff
@@ -14,10 +14,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-:host {
-  overflow-y: auto;
-  display: block;
-  height: 16vw;
+.card {
+  padding: 0px;
+}
+
+.cardTitle {
```

Comment:
Suggestion: Instead of adding a class to title and card component you can use the element as a selector watt-card / watt-card-title

Reply:
Fixed.